### DISCUSSION TITLE
Translate custom properties

### DISCUSSION THREAD

**Discussion by domthomas-dev** [spatie/laravel-medialibrary#2184] [Q&A] [answered]
Hi,

First of all, thank you for this package :)

I can't find how to use spatie / translatable with custom properties.
For a media, I have a custom property for the "alt" property and for the "label" property. And I need to handle multiple languages.

How can I do this?

**Comment by freekmurze** [comment, marked as answer]
Maybe just suffix the custom property with the language code? `name_en`, `name_fr`, ...
